GrandCentralBoard.xcodeproj/project.pbxproj

```diff
@@ -781,6 +795,7 @@
 				1AE1981D1CBE8323003B6AB3 /* UIColor.swift */,
 				1A76DFAD1CBBEF8200A2DF49 /* NSBundle.swift */,
 				F928B82D1CC52F77006D0332 /* String+Localized.swift */,
+				F9E920901CC54BF000BC7994 /* NSDate+StringWithFormat.swift */,
```

Comment:
So for every extension to NSDate we will have to add a file?
We could have them grouped by class they extending ex. `NSDate.swift`.